packages/frontend-api/src/Resources/config/graphql-types/AdvertImageDecorator.types.yaml

```diff
@@ -0,0 +1,23 @@
+AdvertImageDecorator:
+    type: object
+    decorator: true
+    inherits:
+        - 'AdvertDecorator'
+    config:
+        fields:
+            link:
+                type: "String"
+                description: "Advert link"
+            image:
```

Comment:
When I create `Image advert` in administration with empty link and image, I get following error

```
Image of type \"NULL\" not found for entity \"noticer\" with ID 5
```

Reply:
Good catch! Fixed with previous issue in 5b26911377eaae0cb10533f3b7c57aeb84856904